cmd/rook/block/unmap.go

```diff
@@ -29,31 +29,31 @@ import (
 )
 
 var (
-	unmountDeviceName string
-	unmountPath       string
+	unmapDeviceName string
+	unmapPath       string
 )
 
-var unmountCmd = &cobra.Command{
-	Use:   "unmount",
-	Short: "Unmounts and destroys a block device from the local machine",
+var unmapCmd = &cobra.Command{
+	Use:   "unmap",
+	Short: "Unmap and destroys a block device from the local machine",
 }
 
 func init() {
-	unmountCmd.Flags().StringVar(&unmountDeviceName, "device", "", "Name of device to unmount (e.g., rbd0)")
-	unmountCmd.Flags().StringVar(&unmountPath, "path", "", "File system path to unmount")
+	unmapCmd.Flags().StringVar(&unmapDeviceName, "device", "", "Name of device to unmap (e.g., rbd0)")
+	unmapCmd.Flags().StringVar(&unmapPath, "mount", "", "File system mount point to unmount before unmaping")
```

Comment:
I've opened a new issue #506 for enhancing `unmap` to support unmapping by image name.